pilot/pkg/bootstrap/server.go

```diff
@@ -834,6 +852,11 @@ func (s *Server) initRegistryEventHandlers() {
 	}
 }
 
+func onlyStatusUpdated(old config.Config, curr config.Config) bool {
```

Comment:
Is it safe to ignore other metadata? we are only checking part of it

Reply:
some of the other metadata is expected to change, such as resourceVersion.  AFAIK from our conversations, labels and annotations are the only mutable metadata fields which affect pushes.

Reply:
Can you add a comment and/or update the function name to indicate this?